src/environmentd/src/lib.rs

```diff
@@ -128,6 +128,8 @@ pub struct Config {
     // === Special modes. ===
     /// Whether to permit usage of unsafe features.
     pub unsafe_mode: bool,
+    /// What role, if any, should be initially created with elevated privileges.
+    pub bootstrap_role: Option<String>,
```

Comment:
Hmm, this isn't really a "special mode." I'd move this down next to `bootstrap_system_parameters`. (Maybe that section should be renamed from "cloud options" to .. something else, not sure exactly what.)

Reply:
What about moving it to the "Testing Options" section?